dowhy/causal_estimator.py

```diff
@@ -119,15 +129,18 @@ def get_estimator_object(new_data, identified_estimand, estimate):
                 identified_estimand.treatment_variable, identified_estimand.outcome_variable, #names of treatment and outcome
                 test_significance=None,
                 evaluate_effect_strength=False,
-                confidence_intervals = estimate.params["confidence_intervals"],
+                confidence_interval = estimate.params["confidence_intervals"],
```

Comment:
interval, not intervals.

Reply:
I think this may have been responsible for the failure in the refuter.